config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -531,3 +531,33 @@ periodics:
     testgrid-tab-name: kubelet-gce-e2e-swap-fedora
     testgrid-alert-email: ehashman@redhat.com, ikema@google.com
     description: Executes E2E suite with swap enabled on Fedora
+
+- name: ci-kubernetes-node-kubelet-node-performance-test
+  interval: 12h
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20210721-2b77449-master
+        args:
+          - --repo=k8s.io/kubernetes=master
+          - --timeout=90
```

Comment:
Do these timeouts make sense? These tests are failing consistently but I don't know if that has anything to do with the runtime.

Reply:
Should we not have timeouts when tests start passing to make sure that they do not take forever. I think it might be better to let them fail after a time regardless, I do not know what the correct value is though.